docs/csaf-vex.md

```diff
@@ -0,0 +1,497 @@
+objects in the “product_tree” object: `branches` and `relationships`.
+
+#### Branches
+The parent `branches` object has one child object of the `vendor` category with the name set to "Red Hat". All 
+affected Red Hat products and components will be nested in that "branches" array. Compressed down, the parent branches 
+object would look like:
+
+```
+{
+  "branches": [
+    {
+      "branches": []
+      "category": "vendor"
+      "name": "Red Hat:
+    }
+  ]
+}
+```
+
+All nested objects included in the `branches` object of the `vendor` category fall into the following subcategories:
+* `product_family`: Represents a general Red Hat product stream and includes one or more 
+nested objects of the `product_name` category .
+* `product_name`: Represents a specific product release and is always nested under the 
+corresponding "product_family"
+* `product_version`: Represents a specific component. When displayed unnested, the 
+component is not fixed yet and will not include a specific version number. Note: This will only be present in VEX files
+since CSAF files are per advisory and will only include fixed components.
+* `architecture`: Represents fixed components by their architecture and includes nested 
+  "product_version" objects. These "product_version" will be fixed and provide the specific version number. 
+
+
+
+##### Product Family and Product Name Examples
+The `product_family` category represents a general Red Hat product stream and includes one or 
+more nested objects of the `product_name` category that represents an individual release. The `product_name` object will
+always include the name of the product, a product ID and a product identification helper in the form of a CPE. 
+
+In the example below, you can see that the `product_family` object is for Red Hat Enterprise Linux 6 and nested within 
+is the `product_name` object Red Hat Enterprise Linux 6 with the CPE "cpe:/o:redhat:enterprise_linux:6". 
+
+```
+{
+  "branches": [
+    {
+      "category": "product_name",
+      "name": "Red Hat Enterprise Linux 6",
+      "product": {
+        "name": "Red Hat Enterprise Linux 6",
+        "product_id": "red_hat_enterprise_linux_6",
+        "product_identification_helper": {
+          "cpe": "cpe:/o:redhat:enterprise_linux:6"
+        }
+      }
+    }
+  ],
+"category": "product_family",
+"name": "Red Hat Enterprise Linux 6"
+},
+```
+
+##### Unfixed Product Versions (VEX only) Examples
+The `product_version` category includes information about a specific affected package. The `product_version` object will 
+always include the name of the component, a product ID and a product identification helper in the form of a PURL. When 
+displayed unnested under an `architecture` object, the `name` attribute will not reference a specific version number 
+because these components are unfixed. Again, these unfixed `product_version` components will only be found in VEX files 
+since CSAF files always represent a released RHSA.
```

Comment:
RHSA is not mentioned earlier, to it would be good to add a full name here 
Red Hat Security Advisory (RHSA) with a link to https://access.redhat.com/articles/explaining_redhat_errata

Reply:
Will add a link in the CSAF introduction where we first mention advisories